assignments/array-methods.js

```diff
@@ -54,23 +54,28 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
 // ==== Challenge 1: Use .forEach() ====
-// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last namgit pes into a new array called fullName. 
 let fullName = [];
+runners.forEach(element => {
+    fullName.push(element.first_name + " " + element.last_name)  
+});
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
+let allCaps = runners.map((x) => {
+        x.first_name.toUpperCase();
```

Comment:
Great approach to adding the map directly to the `allCaps` variable here when you console log `allCaps` it returns an array filled with undefines. That is because you are not returning your items -> 
```
let allCaps = runners.map((x) => {
        return x.first_name.toUpperCase();
});
```

another way to do it would be pushing the items into an array 
```
let allCaps = []
runners.map((x) => {
        allCaps(x.first_name.toUpperCase())
}
```